### PR TITLE
feat: add conditional configMap and secret volumeMounts to `fullstack-deployment` helm chart

### PR DESCRIPTION
**Pull request by jeromy-cannon**
## Description

This pull request changes the following:

- updates for new location of application.env in ubi-init docker containers
- added configMaps based on values supplied in values.yaml for network-node configuration files
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/config.txt`
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/settings.txt`
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/log4j2.xml`
  - file: `/etc/network-node/application.env`
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/data/config/api-permission.properties`
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/data/config/application.properties`
  - file: `/opt/hgcapp/services-hedera/HapiApp2.0/data/config/bootstrap.properties`
- added leverage of secrets if they exist:
  - file `/opt/hgcapp/services-hedera/HapiApp2.0/hedera.crt` 
  - file `/opt/hgcapp/services-hedera/HapiApp2.0/hedera.key`
  - directory: `/opt/hgcapp/services-hedera/HapiApp2.0/data/keys`
  
### Related Issues

- Required by https://github.com/hashgraph/solo/issues/110
- Required by https://github.com/hashgraph/solo/issues/136
- Required by https://github.com/hashgraph/solo/issues/135
- Required by https://github.com/hashgraph/solo/issues/134

### CODE REVIEW

File: charts/fullstack-deployment/templates/configmaps/network-node-etc-cm.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{- if $.Values.hedera.configMaps.applicationEnv }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-node-etc-cm
+  namespace: {{ default $.Release.Namespace $.Values.global.namespaceOverride }}
+  labels:
+    {{- include "fullstack.testLabels" $ | nindent 4 }}
+data:
+  applicationEnv: |
```
Comment (nathanklick) [inline charts/fullstack-deployment/templates/configmaps/network-node-etc-cm.yaml:10]: We should use the filename instead of the values variable name. Not urgent, may be done as a separate PR.

File: charts/fullstack-deployment/templates/configmaps/network-node-hapi-app-cm.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-node-hapi-app-cm
+  namespace: {{ default $.Release.Namespace $.Values.global.namespaceOverride }}
+  labels:
+    {{- include "fullstack.testLabels" $ | nindent 4 }}
+data:
+{{- if $.Values.hedera.configMaps.configTxt }}
+  configTxt: |
```
Comment (nathanklick) [inline charts/fullstack-deployment/templates/configmaps/network-node-hapi-app-cm.yaml:11]: We should use the filename instead of the values variable name. Not urgent, may be done as a separate PR.

**Review by nathanklick — APPROVED** [review]

File: charts/fullstack-deployment/templates/configmaps/network-node-etc-cm.yaml (hunk 1 above)
Comment (jeromy-cannon) [reply charts/fullstack-deployment/templates/configmaps/network-node-etc-cm.yaml:10]: https://github.com/hashgraph/full-stack-testing/issues/844

File: charts/fullstack-deployment/templates/configmaps/network-node-hapi-app-cm.yaml (hunk 2 above)
Comment (jeromy-cannon) [reply charts/fullstack-deployment/templates/configmaps/network-node-hapi-app-cm.yaml:11]: https://github.com/hashgraph/full-stack-testing/issues/844
